elks/arch/i86/drivers/char/console-font-8x8.S

```diff
@@ -0,0 +1,174 @@
+	.section .text
+	.global font_8x8
+font_8x8:
+	.byte 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x7E,0x81,0xA5,0x81
```

Comment:
Can you rename this file console-font-8x8-swan.S? That will make it very apparent where it's used.

Reply:
My initial thinking was that future "graphics mode as text mode" drivers could use the same file, making it a bit more generic; but sure.